iml-agent/src/action_plugins/lustre/snapshot.rs

```diff
@@ -45,3 +45,29 @@ pub async fn destroy(d: Destroy) -> Result<(), ImlAgentError> {
     }
     lctl(args).await.map(drop)
 }
+
+pub async fn mount(
+    (filesystem_name, snapshot_name): (String, String),
+) -> Result<(), ImlAgentError> {
+    let args = &[
+        "snapshot_mount",
+        "--fsname",
+        &filesystem_name,
+        "--name",
+        &snapshot_name,
+    ];
+    lctl(args).await.map(drop)
+}
+
+pub async fn unmount(
+    (filesystem_name, snapshot_name): (String, String),
+) -> Result<(), ImlAgentError> {
+    let args = &[
+        "snapshot_umount",
```

Comment:
I wonder if we should stick to the lctl naming (umount/unmount) :)

Reply:
I don't want these Unixy 1-letter contractions in our code to be honest